### PR TITLE
[WIP] New filters

### PR DESCRIPTION
**Pull request by Xunrel**
Hi,

I wanted to give it a try and contribute something.
I adjusted the ColumnHelper.js to include a restriction filter for all pages where includeRestricted is true.

I'm currently also trying to look at a possible solution for the price filtering.

Regards
Daniel

### CODE REVIEW

**Comment by Xunrel** [comment]
Okay, as I see it fails because of the equality test.
I rewrote them and now it should be passing.

**Comment by duffn** [comment]
Thanks for the PR!

Can you make it clearer that the filter has to do with the price. "Restricted Price" or whatever you see fit.

File: src/components/shared/ColumnHelper.js [hunk 1]
```diff
@@ -4,6 +4,27 @@ import Link from "../shared/Link"
 export const GENERATED_ID_COL_INDEX = 0
 export const RESTRICTED_COL_INDEX = 1
 
+const RESTRICTED_FILTER_NAMES = ["Restricted", "Not restricted"];
+export const RESTRICTED_FILTER = {
+  label: "Restricted",
```
Comment (duffn) [inline src/components/shared/ColumnHelper.js]: Can you make this "Restricted Price" or something along those lines so it's clear that this is for the price?

Comment (Xunrel) [reply src/components/shared/ColumnHelper.js]: Sure, no problem. I changed it to "RESTRICTED_PRICE_FILTER" and changed the labl to "Restricted price"

**Comment by duffn** [comment]
Thanks for the contribution!

**Pull request merged**